mirdata/utils.py

```diff
@@ -184,6 +177,17 @@ def update_to(self, b=1, bsize=1, tsize=None):
         self.update(b * bsize - self.n)
 
 
+def download_large_file(url, download_path, callback=None):
+    response = requests.get(url)
+    response.raise_for_status()
+    with open(download_path, 'wb') as handle:
+        for block in response.iter_content(1024):
+            handle.write(block)
+            if callback:
```

Comment:
Could just default callback to lambda : pass. Is there any reason why we need this here. Is anyone using callbacks?

Reply:
Generally nice to have callbacks available in anything that is async, although I am not entirely sure how async is handled in python (i.e. is there a way to return a promise?). I think this is fine, in case someone somewhere does want it.

Reply:
@drubinstein, I put it there so we could update `DownloadProgressBar` with each chunk!  Happy to take it out, myself

@mmscibor, this is not async - the callback fires when each block is written.

Reply:
That... Is a good reason! Keep it. I still believe in my comment about the default.